openbsd-compat/arc4random.c

```diff
@@ -113,8 +113,13 @@ _rs_stir(void)
 		fatal("Couldn't obtain random bytes (error 0x%lx)",
 		    (unsigned long)ERR_get_error());
 #else
+#ifdef WINDOWS
```

Comment:
Please put a TODO comment here to take care of replacing rand() with a more secure generator. You may look at other TODOs in the sources